### PR TITLE
Sort UTXO in stakeable txs

### PR DESCRIPTION
**Pull request by cgcardona**
If this is a stakeable transaction then have `StakeableLockOut` come before `SECPTransferOutput` so that users first stake locked tokens before staking unlocked tokens. Also sort the `StakeableLockOuts` by `StakeableLocktime` so that the greatest `StakeableLocktime` are spent first

### CODE REVIEW

**Comment by sezu** [comment]
Can we get this merged and a new release cut? Since the types aren't currently sorted, you can end up in a position where you need change for both `LockedStakeable` and `Unlocked` funds. Unfortunately, the code expects there to only be a single change so transactions are currently failing.

example: L: locked, U: unlocked
stake amount: 10
ins: [L5, U3, L6]
expected change: [L1, U3]
actual change: [L4]

**Comment by cgcardona** [comment]
@sezu yes the team is QAing this now and will go out in the next AvalancheJS release. Hopefully by EOW. I'll msg you here once it's live. Thanks! 🎩

**Comment by dasconnor** [comment]
The code itself looks good. Would it be possible to add a unit test?

File: tests/apis/platformvm/api.test.ts [hunk 1]
```diff
@@ -1232,6 +1234,341 @@ describe('PlatformVMAPI', () => {
...
+        addrs1, 
+        addrs2, 
+        nodeID, 
+        startTime,
+        endTime,
+        stakeAmount,
+        addrs3, 
+        delegationFeeRate
+      );
+      console.log(txu1.toBuffer().toString('hex'))
```
Comment (dasconnor) [inline tests/apis/platformvm/api.test.ts:1509]: You have a bunch of commented out console.logs and this live one.

**Review by dasconnor — APPROVED** [review]
One small comment about some lingering print statements. Otherwise looks good to me!

**Comment by cgcardona** [comment]
@sezu this is live in AvalancheJS v3.0.6.

**Pull request merged**